packages/asc-ui/src/components/TopTaskButton/TopTaskButton.stories.tsx

```diff
@@ -0,0 +1,55 @@
+import { Location, Passport } from '@datapunt/asc-assets'
+import React from 'react'
+import Readme from './README.md'
+import TopTaskButton from './TopTaskButton'
+
+export default {
+  title: 'Experimental/Atoms/TopTaskButton',
+  parameters: {
+    notes: Readme,
+  },
+  decorators: [
+    (storyFn: () => React.ReactNode) => (
+      <div
+        style={{
+          display: 'flex',
+          maxWidth: 400,
+          flexDirection: 'column',
+          alignItems: 'flex-start',
+          padding: '40px 10px',
+        }}
+      >
+        {storyFn()}
+      </div>
+    ),
+  ],
+}
+
+export const WithIcon = () => (
+  <TopTaskButton
+    href="https://amsterdam.nl"
+    icon={Location}
+    text="Stadsloket: locaties en openingstijden"
+    style={{ maxWidth: 220 }}
+  />
+)
+
+export const WithoutIcon = () => (
+  <TopTaskButton
+    href="https://amsterdam.nl"
+    text="Stadsloket: locaties en openingstijden"
+    style={{ maxWidth: 220 }}
+  />
+)
+
+export const OnDarkBackground = () => (
```

Comment:
Can we add an example with an onclick event?

Reply:
The intention is that this should always be an anchor element, since this is a navigational element.

As far as the href is concerned it would just be an empty link right? Which is something you can also do in plain HTML, even if it's not the best idea.